pkgs/development/misc/haskell/hasura/graphql-engine/default.nix

```diff
@@ -2,11 +2,11 @@
 , asn1-types, async, attoparsec, attoparsec-iso8601, auto-update
 , base, base64-bytestring, byteorder, bytestring, case-insensitive
 , ci-info, containers, criterion, cryptonite, data-has, deepseq
-, dependent-map, dependent-sum, directory, ekg-core, ekg-json
+, dependent-map_0_2_4_0, dependent-sum_0_4, directory, ekg-core, ekg-json
 , fast-logger, fetchgit, file-embed, filepath, generic-arbitrary
 , ghc-heap-view, graphql-parser, hashable, hspec, hspec-core
 , hspec-expectations-lifted, http-client, http-client-tls
-, http-types, immortal, insert-ordered-containers, jose, lens
+, http-types, immortal_0_2_2_1, insert-ordered-containers, jose, lens
```

Comment:
It would be much more recommended to make a clean split here.
This file should just be the output generated by cabal2nix.

All manual overrides should happen in `pkgs/development/haskell-modules/configuration-common.nix`. That way the next time you update this file you don‘t need to manually insert version overrides again.